<launch>
    <!-- hanp head behavior -->
    <node pkg="hanp_head_behavior" type="hanp_head_behavior" name="hanp_head_behavior" output="screen">
        <param name="local_plan_sub_topic" value="/move_base_node/HANPLocalPlanner/local_plan"/>
        <param name="robot_base_frame" value="base_link"/>
        <param name="human_sub_topic" value="/humans"/>

        <param name="aliveness_weight" value="0.9"/>
        <param name="interaction_weight" value="1.0"/>
        <param name="social_attention_weight" value="1.0"/>
        <param name="intent_projection_weight" value="1.0"/>

        <param name="point_head_height" value="1.1"/>
        <param name="max_gma" value="1.05" />
        <param name="visibility_angle" value="0.1"/>

        <param name="local_plan_end_extend" value="0.7"/>
    </node>
</launch>
